lxd/backup/backup_instance.go

```diff
@@ -78,6 +79,16 @@ func (b *InstanceBackup) Rename(newName string) error {
 		}
 	}
 
+	// Defend against path traversal attacks.
```

Comment:
rather than checking the `name` is safe by checking the resulting paths, is the reason you've not validated the `newName` itself and then used the sanitised version of that to build the paths?